host/cli/update.go

```diff
@@ -16,17 +19,21 @@ import (
 	"github.com/flynn/flynn/pinkerton/layer"
 	"github.com/flynn/flynn/pkg/cluster"
 	"github.com/flynn/flynn/pkg/exec"
+	"github.com/flynn/flynn/pkg/tufutil"
 )
 
 func init() {
 	Register("update", runUpdate, `
-usage: flynn-host update [--driver=<name>] [--root=<path>] [--repository=<uri>] [--tuf-db=<path>]
+usage: flynn-host update [options]
 
 Options:
   -d --driver=<name>       image storage driver [default: aufs]
   -r --root=<path>         image storage root [default: /var/lib/docker]
   -u --repository=<uri>    image repository URI [default: https://dl.flynn.io/tuf]
   -t --tuf-db=<path>       local TUF file [default: /etc/flynn/tuf.db]
+  -b --bin-dir=<dir>       directory to download binaries to [default: /usr/local/bin]
+  -c --config-dir=<dir>    directory to download config files to [default: /etc/flynn]
+  --is-latest              internal flag (skip updating local tuf DB and re-execing latest binary)
```

Comment:
It looks like if this flag is specified, it will delete the binary that is being run from. This is not obvious, perhaps use a separate flag to cause the removal?